src/doc/data_into_visit/VTKFormat.rst

```diff
@@ -0,0 +1,540 @@
+
+Cell data
+~~~~~~~~~
+
+The cell data section starts with a single line with the keyword `CELL_DATA` followed by the number of cells.
+The number of cells must match the number of cells defined by the topology.
+Next comes a list of fields.
+Fields can be either scalars, vectors or tensors.
+
+Scalar fields
+"""""""""""""
+
+The scalar field section has the following structure. ::
+
+    SCALARS fieldName dataType 1
+    LOOKUP_TABLE default
+    s1 s2 ... sN
+
+The scalar field section starts with a single line with the keyword `SCALARS` followed by the name of the field followed by the data type followed by the number of values per scalar.
+The number of values per scalar is optional, and if present, must be `1`.
+Next comes the lookup table information.
+The lookup table information consists of a single line with the keyword `LOOKUP_TABLE` followed by the keyword `default`.
+The lookup table specifies the lookup table name when mapping the scalar to colors.
+The lookup table is not used by VisIt_.
+Next come the scalar values.
+The scalar values can be split up into lines in an arbitrary manner.
+
+Vector fields
+"""""""""""""
+
+The vector field section has the following structure. ::
+
+    VECTORS fieldName dataType
+    v11 v12 v13
+    v21 v22 v23
+        ...
+    vN1 vN2 vN3
+
+The vector field section starts with a single line with the keyword `VECTORS` followed by the name of the field followed by the data type.
+Next come the vector values.
+The vector values consist of three values per point, regardless of whether it is for a 2D or 3D mesh.
+The vector values can be split up into lines in an arbitrary manner.
+
+Tensor fields
+"""""""""""""
+
+The tensor field section has the following structure. ::
+
+    TENSORS fieldName dataType
+    t111 t112 t113
+    t121 t122 t123
+    t131 t132 t133
+    t211 t212 t213
+    t221 t222 t223
+    t231 t232 t233
+        ...
+    tN11 tN12 tN13
+    tN21 tN22 tN23
+    tN31 tN32 tN33
+
+The tensor field section starts with a single line with the keyword `TENSORS` followed by the name of the field followed by the data type.
+Next come the tensor values.
+The tensor values consist of nine values per point, regardless of whether it is for a 2D or 3D mesh.
+The tensor values can be split up into lines in an arbitrary manner.
+
+Point data
+~~~~~~~~~~
+
+The point data section starts with a single line with the keyword `POINT_DATA` followed by the number of points.
+The number of points must match the number of points defined by the topology.
+The rest of the point data section is the same as the cell data section.
+
+.. _data_into_visit_vtk_conventions:
+
+VisIt_ meta data conventions for VTK files
+------------------------------------------
+
+VisIt_ supports a number of conventions for storing additional data. This data is stored as FIELD data as additional information in the DATASET, CELL_DATA or POINT_DATA.
+
+The Following meta data is stored as FIELD data.
+
+MeshName
```

Comment:
I just ran across another VisIt convention, MeshCoordType, as described in this [discussion](https://github.com/visit-dav/visit/discussions/13196#discussioncomment-1126021) for specifying the mesh is non-cartesian.

Reply:
Fixed.